Competitive_Programming/Python/Stacks & Queues/README.md

```diff
@@ -0,0 +1,382 @@
+# Stacks and Queues
+
+Stacks and queues are similar data structures difference being only in the way of how the elements are removed.
+
+# Stacks
+
+## Introduction
+
+A **stack** is a LIFO (Last In First Out) data structure. A stack pushes (inserts) a new element to the top of the stack and also pops (removes) the element from the top.
+
+<br>
+
+<img src="assets/stack.png"  style="max-width: 80%; height: auto; background:white"/>

```

Comment:
add images via link instead of attaching. You can upload it to any issue and you'll get the link. And remove image from assets.